src/instruments/src/Common/displayUnit.tsx

```diff
@@ -53,10 +59,9 @@ export const DisplayUnit: React.FC<DisplayUnitProps> = (props) => {
             setTimer(null);
         }
     });
-
-    if (state === DisplayUnitState.Selftest) {
-        return (
-            <svg className="SelfTest" viewBox="0 0 600 600">
+    return (
```

Comment:
I'm not sure if the previous code was intentionally emitting less DOM elements. @lukecologne  @Benjozork does either of you have an opinion on this?

Reply:
Pretty much necessary to avoid an issue where the useUpdate hook would unhook itself from the event listener when loading in in-flight.